src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,111 @@
+import calculate from '../logic/calculate';
+
+describe('Start tests', () => {
+  test('8 - 4 = 4', () => {
+    let item = {};
+
+    item = { ...item, ...calculate(item, '8') };
+    item = { ...item, ...calculate(item, '-') };
+    item = { ...item, ...calculate(item, '4') };
+    item = { ...item, ...calculate(item, '=') };
+
+    expect(item).toEqual({
+      total: '4',
+      next: null,
+      operation: null,
+    });
+  });
```

Comment:
- [ ] This test suite is failing, please fix this test and submit it for another review.

![tttt](https://user-images.githubusercontent.com/61562626/135711125-2bd6ca07-a7d4-4e35-95fa-90d42bedce32.png)